### PR TITLE
feat: add features page for erc721

### PR DESCRIPTION
**Pull request by jenyafomin**

### CODE REVIEW

**Comment by jenyafomin** [comment]
Я там добавил два файла features & features1. В features поменьше текста в features1 по больше.
Там думаю еще нужно будет добавить какие-то скриншоты к каждому из пунктов и возможно что-то объеденить из этих двух файлов. Они в целом похожи но написаны чуть по разному.

File: docs/admin-panel/hierarchy/ERC721/contract.md [hunk 1]
```diff
@@ -15,7 +15,13 @@ ERC721 contracts are being deployed using [Contract Manager](/docs/admin-panel/m
 This information is passed directly to blockchain
 
 - **template** - Smart contract template with its unique functions
-    - **SIMPLE** - Simple template includes all basic functions like _mint_, _burn_ and _transfer_
+    - **SIMPLE** - Simple template includes all basic functions like ***mint***, ***burn*** and ***transfer***. Consist of several contracts:
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/contract.md:18]: and several extensions

File: docs/admin-panel/hierarchy/ERC721/contract.md [hunk 2]
```diff
@@ -15,7 +15,13 @@ ERC721 contracts are being deployed using [Contract Manager](/docs/admin-panel/m
...
+      - **Access Control** - provides the ability to restrict or grant permission for specific addresses to interact with a smart contract. (see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/access-control))
+      - **Burnable** -  allows for a token to be destroyed, reducing the overall supply and potentially increasing the value of remaining tokens (see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Burnable))
+      - **Enumerable** - allows the contract to be queried for a list of all token holders and their token balances (see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Enumerable))
+      - **Royalty** - allows for a portion of the sale price to be paid to the original creator of the NFT each time it is resold
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/contract.md:22]: link to https://eips.ethereum.org/EIPS/eip-4907

File: docs/admin-panel/hierarchy/ERC721/contract.md [hunk 3]
```diff
@@ -15,7 +15,13 @@ ERC721 contracts are being deployed using [Contract Manager](/docs/admin-panel/m
...
+      - **BaseUrl** - allows the NFTs to have a URI pointing to a public location where additional information and resources can be stored
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/contract.md:23]: pointing to out JSON (link to json) microservice

File: docs/admin-panel/hierarchy/ERC721/contract.md [hunk 4]
```diff
@@ -15,7 +15,13 @@ ERC721 contracts are being deployed using [Contract Manager](/docs/admin-panel/m
...
+      - **MetaData Getter** - <u>...</u>
     - **GRADED** - Extends **SIMPLE** plus _metadata storage_, which can be used for updating level of the item
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/contract.md:25]: nope, metadata is already in simple to store templateId
Graded just has functions to update it

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+### SIMPLE
+
+The SIMPLE contract is a foundational component of all features offered by our platform. It includes all the basic functions necessary for the creation and management NFTs, such as **minting**, **burning**, and **transferring**.
+
+#### Consist of several contracts:
+- **Access Control** - provides the ability to restrict or grant permission for specific addresses to interact with a smart contract. _(see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/access-control))_
+- **Burnable** -  allows for a token to be destroyed, reducing the overall supply and potentially increasing the value of remaining tokens _(see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Burnable))_
+- **Enumerable** - allows the contract to be queried for a list of all token holders and their token balances _(see [OpenZeppelin](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721#ERC721Enumerable))_
+- **Royalty** - allows for a portion of the sale price to be paid to the original creator of the NFT each time it is resold
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:16]: ссылка на ERC2981

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+
+### BLACKLIST
+
+The **Blacklist** feature is a security mechanism that **restrict** certain addresses from **transfering NFT**. These addresses are typically considered to be bad actors and are unable to use the NFTs they possess if they are added to blacklist. This helps to maintain the integrity of the contract and ensure that NFTs are not being used maliciously.
+
+
+### WHITELIST
+
+The **Whitelist** feature in NFTs works as the opposite of the Blacklist. While the blacklist restrict specific addresses from transfering the NFT, the Whitelist **allows only a specific set of addresses transfer the NFT**.
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:30]: opposite of the **Blacklist** 

bold

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+
+### PAUSABLE
+
+The **Pausable** feature in smart contracts allows the contract owner or authorized parties to **temporarily pause the execution of the contract**. This feature is particularly useful when the marketing team starts selling NFTs, but the programming side is not yet ready to support the transaction process. It can also be used to address unexpected bugs or errors that may arise during the lifetime of the contract, without affecting the overall integrity of the contract.
+
+One practical example of its use is in the sale of mystery boxes. If the NFTs within the mystery boxes are not yet available, the contract can be paused until they are ready. This ensures that the users cannot access the NFTs until the contract is ready, adding an extra layer of security and control to the process. The Pausable feature of ERC721 contracts allows for greater flexibility and control, making it an important component for any project utilizing NFTs.
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:37]: adding an extra layer of security and control to the process
->
allowing the marketing team to win time for developers or something like this

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+
+### UPGRADEABLE
+
+The **Upgradeable** feature allows NFTs to be upgraded with attributes such as level, power, stamina, etc. It provides a new level of functionality to NFTs by allowing owners to enhance their assets. This can increase the value and desirability of NFTs, making them more appealing to buyers and investors.
+
+In a gaming platform, for example, players can upgrade their NFTs to improve their in-game abilities or earn more in-game currency. This not only enhances the user's experience but also positively influences the overall economy of the token as upgraded tokens may hold a higher value.
+
+
+### GENES
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:47]: put genes after random

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+
+### RANDOM
+
+The **RANDOM** feature is an important aspect of the ERC721 contract, as it provides a way to randomly generate events within a blockchain application. To ensure the fairness and transparency of the random number generation process, we use the popular decentralized oracle service, [ChainLink](/docs/integrations/chain-link/).
+
+Combining the RANDOM feature with the UPGRADABLE feature also provides new possibilities for NFT-based gaming experiences. Players can be surprised by the rarity of the NFT they receive and can then work to upgrade it and increase its value. This creates a unique and dynamic NFT economy, where the value of NFTs is determined by both player effort and luck.
+
+You can also combine RANDOM with other features such as MYSTERY BOX, and this can enhance the overall user experience and drive engagement. The use of RANDOM in NFT projects, such as in gaming, can greatly add to the value and desirability of the NFTs, leading to a more active and thriving community and economy.
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:58]: have to change wording to explicitly say that random nft is inside lootbox

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+### SOULBOUND
+
+The **SOULBOUND** feature is a valuable addition to NFTs, providing security and control to the distribution and ownership of these tokens. By tying ownership to a specific individual, the SOULBOUND feature ensures that the NFT remains in the hands of its intended owner, rather than being traded or passed on to others. This is particularly useful for NFTs with important or high-value attributes, such as **Governance tokens** used for voting purposes.
+
+The use of the SOULBOUND feature as a voting pass in [governance](/docs/admin-panel/complex-mechanics/governance/) systems ensures that each vote is **securely tied** to an individual voter, preventing fraudulent or duplicated voting. This provides a level of trust and accountability to the [governance](/docs/admin-panel/complex-mechanics/governance/) process, ensuring that the outcome accurately reflects the will of the voters.
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:64]: soulbound can be separated from voting. have to add one more feature VOTING

**Review by TrejGun — CHANGES_REQUESTED** [review]

File: docs/admin-panel/hierarchy/ERC721/features.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+
+
+### RANDOM
+
+The **RANDOM** feature is an important aspect of the ERC721 contract, as it provides a way to randomly generate events within a blockchain application. To ensure the fairness and transparency of the random number generation process, we use the popular decentralized oracle service, [ChainLink](/docs/integrations/chain-link/).
+
+Combining the RANDOM feature with the UPGRADABLE feature also provides new possibilities for NFT-based gaming experiences. Players can be surprised by the rarity of the NFT they receive and can then work to upgrade it and increase its value. This creates a unique and dynamic NFT economy, where the value of NFTs is determined by both player effort and luck.
+
+You can also combine RANDOM with other features such as MYSTERY BOX, and this can enhance the overall user experience and drive engagement. The use of RANDOM in NFT projects, such as in gaming, can greatly add to the value and desirability of the NFTs, leading to a more active and thriving community and economy.
+
```
Comment (TrejGun) [inline docs/admin-panel/hierarchy/ERC721/features.md:59]: there is one more feature RENTABLE which indicates the presence of the ERC4907 extension

**Review by TrejGun — CHANGES_REQUESTED** [review]

File: docs/admin-panel/hierarchy/ERC721/contract.md (hunk 1 above)
Comment (jenyafomin) [reply docs/admin-panel/hierarchy/ERC721/contract.md]: Already different structure. I don't mention about any features in contract page.
I will add this change in features page

File: docs/admin-panel/hierarchy/ERC721/contract.md (hunk 4 above)
Comment (jenyafomin) [reply docs/admin-panel/hierarchy/ERC721/contract.md]: Old code, right now don't have it

**Pull request merged**